modules/nw-aws-replacing-clb-with-nlb.adoc

```diff
@@ -4,18 +4,22 @@
 
 :_content-type: PROCEDURE
 [id="nw-aws-replacing-clb-with-nlb_{context}"]
-= Replacing Ingress Controller Classic Load Balancer with Network Load Balancer 
+= Replacing Ingress Controller Classic Load Balancer with Network Load Balancer
 
-You can replace an Ingress Controller that is using a Classic Load Balancer (CLB) with one that uses a Network Load Balancer (NLB) on AWS. 
+You can replace an Ingress Controller that is using a Classic Load Balancer (CLB) with one that uses a Network Load Balancer (NLB) on AWS.
 
 [WARNING]
 ====
-This procedure causes an expected outage that can last several minutes due to new DNS records propagation, new load balancers provisioning, and other factors. IP addresses and canonical names of the Ingress Controller load balancer might change after applying this procedure. 
+This procedure might cause the following issues:
+
+* An outage that can last several minutes due to new DNS records propagation, new load balancers provisioning, and other factors. IP addresses and canonical names of the Ingress Controller load balancer might change after applying this procedure.
```

Comment:
Should we change `can` to `might` since  it would follow along with the use of `might` like mentioned in the above "This procedure might cause the following issues:" Up to you if you would like to use. I see `can` is an okay word to use based on the IBM style guide https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m (under may)